account/user_blackbox_test.go

```diff
@@ -58,6 +59,49 @@ func (s *userBlackBoxTest) TestOKToLoad() {
 	createAndLoadUser(s) // this function does the needful already
 }
 
+func (s *userBlackBoxTest) TestLoadByUsername() {
+	t := s.T()
+	resource.Require(t, resource.Database)
+	t.Run("load ok", func(t *testing.T) {
+		fxt := tf.NewTestFixture(s.T(), s.DB, tf.Identities(1, func(fixture *tf.TestFixture, idx int) error {
```

Comment:
You must use `t` instead of `s.T()` because `s.T()` refers to the top level test and not the nested one.

```suggestion
		fxt := tf.NewTestFixture(t, s.DB, tf.Identities(1, func(fixture *tf.TestFixture, idx int) error {
```